docker-compose.yaml

```diff
@@ -107,6 +107,6 @@ services:
     ports:
       - ${KEYCLOAK_PORT}:8080
     volumes:
-      - ./keycloak/scripts:/usr/scripts
-      - ./keycloak/scripts/disable-theme-cache.cli:/opt/jboss/startup-scripts/disable-theme-cache.cli
+      - ./keycloak/bin:/usr/local/bin
+      - ./keycloak/startup-scripts:/opt/jboss/startup-scripts
       - ./keycloak/theme/ssmp:/opt/jboss/keycloak/themes/ssmp
```

Comment:
We might as well change the folder name from `ssmp` -> `osmp`

Reply:
I was afraid that might mess with the theme for the realm actually in use.

Reply:
It should be fine. The only thing that would need to change with this is in the `osmp/keycloak/scripts/bootstrap-keycloak.sh`, we'd need to change the theme name from `ssmp` to `osmp`

Reply:
Then, what I'd do is `docker-compose up --force-recreate` and then run the bootstrap script again

Reply:
We're actually not using the same sets of command for loading theme in staging, so it doesn't get affected there